frontend/packages/ux-editor/src/components/FormComponent/FormComponent.module.css

```diff
@@ -36,9 +36,15 @@
 }
 
 .editMode .dragHandle,
-.editMode .buttons,
+.editMode .buttons {
+  visibility: visible;
+}
+
 .wrapper:hover .dragHandle,
-.wrapper:hover .buttons,
+.wrapper:hover .buttons {
+  visibility: visible;
+}
+
```

Comment:
Har disse endringene noe for seg?

Reply:
Jeg fikk feil i Cypress test,  derfer gjorde jeg de endringene her, skal sette alt tilbake siden at komponanten skal fjernes.